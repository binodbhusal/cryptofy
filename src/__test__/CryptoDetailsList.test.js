import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../__MockTest__/mockstore';
import CryptoDetailList from '../Component/CryptoDetailsList';

describe('CryptoDetailList component', () => {
  test('renders the selected crypto detail', () => {
    const { container } = render(
      <Provider store={store}>
        <MemoryRouter initialEntries={['/crypto-details/binance']}>
          <CryptoDetailList />
        </MemoryRouter>
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
