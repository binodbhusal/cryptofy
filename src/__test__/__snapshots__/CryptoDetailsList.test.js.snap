// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CryptoDetailList component renders the selected crypto detail 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
